BNomad/View/SignUpView/SignUpViewController.swift

```diff
@@ -409,9 +505,29 @@ class SignUpViewController: UIViewController {
                 requestLabel.asColor(targetString: requestItem[index], color: CustomColor.nomadBlue ?? .label)
             }
             
+        } else if introductionField.isHidden == false && profileImageButton.isHidden == true {
+            if introductionField.text.isEmpty == true || introductionField.text == introPlaceholder {
+                print("자기소개를 입력하세요.")
+            } else {
+                profileImageButton.isHidden = false
+                plusView.isHidden = false
+                dot3View.backgroundColor = CustomColor.nomadGray2
+                dot4View.backgroundColor = CustomColor.nomadBlue
+                inputConfirmButton.setTitle("확인", for: .normal)
+                
+                index = 3
+                updateRequestLabel(index: index)
+                requestLabel.asColor(targetString: requestItem[index], color: CustomColor.nomadBlue ?? .label)
+            }
         } else {
-            if let nickname = nicknameField.text, let occupation = occupationField.text, let intro = statusField.text {
-                if nickname.isEmpty == false && occupation.isEmpty == false && intro.isEmpty == false {
+            if let nickname = nicknameField.text, let occupation = occupationField.text, let intro = introductionField.text {
+                var isIntroDone = false
+                if intro != introPlaceholder && intro.isEmpty == false {
+                    isIntroDone = true
+                }
+                
+                if nickname.isEmpty == false && occupation.isEmpty == false && isIntroDone == true {
```

Comment:
```suggestion
                if !nickname.isEmpty && !occupation.isEmpty && isIntroDone == true {
```
[P5] `isEmpty`라는게 이미 bool값을 알려주고 있는 것 같아 위에처럼 간략히 축약도 좋을 것 같습니다👍

Reply:
수정했습니다! 또 하나 배웠네요..